frontend/src/old-pages/Configure/Cluster.tsx

```diff
@@ -404,40 +415,44 @@ function Cluster() {
     isMultipleInstanceTypesActive,
   ])
 
+  const handleMultiUserChange: NonCancelableEventHandler<
+    CheckboxProps.ChangeDetail
+  > = ({detail}) => {
+    if (!detail.checked) {
+      clearState(['app', 'wizard', 'config', 'DirectoryService'])
+    }
+    setState(['app', 'wizard', 'multiUser'], detail.checked)
+  }
+
   return (
-    <Container>
-      <SpaceBetween direction="vertical" size="s">
-        <RegionSelect />
-        <OsSelect />
-        <VpcSelect />
-        {isMultiuserClusterActive && (
-          <FormField>
-            <Header
-              // @ts-expect-error TS(2322) FIXME: Type '"h4"' is not assignable to type 'Variant | u... Remove this comment to see the full error message
-              variant="h4"
-              description={t('wizard.cluster.multiUser.description')}
-            >
-              <Trans i18nKey="wizard.cluster.multiUser.title" />
-            </Header>
-            <Toggle
-              disabled={editing}
-              checked={multiUserEnabled}
-              onChange={() =>
-                setState(['app', 'wizard', 'multiUser'], !multiUserEnabled)
-              }
-            >
-              <Trans i18nKey="wizard.cluster.multiUser.label" />
-            </Toggle>
-          </FormField>
-        )}
-        <CustomAMISettings
-          basePath={configPath}
-          appPath={['app', 'wizard']}
-          errorsPath={errorsPath}
-          validate={clusterValidate}
-        />
-      </SpaceBetween>
-    </Container>
+    <SpaceBetween direction="vertical" size="s">
+      <Container>
+        <SpaceBetween direction="vertical" size="s">
+          <RegionSelect />
+          <OsSelect />
+          <VpcSelect />
+          <CustomAMISettings
+            basePath={configPath}
+            appPath={['app', 'wizard']}
```

Comment:
suggestion(non-blocking): this array `['app', 'wizard']` can easily be put outside the re-rendered template. This way you would pass the same reference avoiding useless re-renders